Model/Button/GetCartButtonConfig.php

```diff
@@ -75,7 +74,16 @@ private function addRegularCheckoutButtonConfig(AmwalButtonConfigInterface $butt
         $buttonConfig->setAddressRequired(false);
         $buttonConfig->setInitialAddress($formatedAddress ?? null);
         $buttonConfig->setInitialEmail($email);
-        $buttonConfig->setInitialPhone($shippingAddress->getTelephone() ?? null);
+        if (class_exists('libphonenumber\PhoneNumberUtil')) {
+            $phoneNumberUtil = PhoneNumberUtil::getInstance();
+            try {
+                $swissNumberProto = $phoneNumberUtil->parse($shippingAddress->getTelephone(), $shippingAddress->getCountryId());
+                $phone_number = $phoneNumberUtil->format($swissNumberProto, \libphonenumber\PhoneNumberFormat::E164);
+            } catch (\libphonenumber\NumberParseException $e) {}
```

Comment:
if an error happens, `$phone_number` will not be defined. I would suggest moving the else part before the if statement as fall back. something like this
```
$phone_number = $shippingAddress->getTelephone();
if (class_exists('libphonenumber\PhoneNumberUtil')) {
            $phoneNumberUtil = PhoneNumberUtil::getInstance();
            try {
                $parsed_number = $phoneNumberUtil->parse($phone_number, $shippingAddress->getCountryId());
                $phone_number = $phoneNumberUtil->format($parsed_number, \libphonenumber\PhoneNumberFormat::E164);
            } catch (\libphonenumber\NumberParseException $e) {}
}
```

Reply:
Ok, she cover from the condition else, but np i will do your suggest